docs/version0.55.md

```diff
@@ -45,6 +46,20 @@ A new system property, `-Djava.security.propertiesList` is added to configure a
 
 For more information, see [`-Djava.security.propertiesList`](djavasecuritypropertieslist.md).
 
+### The format of the `java.vm.version` system property value is updated to be compatible with the Runtime.Version parser
+
+The format of the `java.vm.version` system property value is modified from the previous releases. The new format of the value is standardized and structured and is parse-able by the `java.lang.Runtime.Version` class. The OpenJ9 version is at the end of the optional component. This modification also changes the `-version` output, whereby `build openj9-0.55.0` changes to `build 25+36-openj9-0.55.0`.
```

Comment:
The -version output `build openj9-0.55.0` is an example, the text should make that clear.

Reply:
The OpenJ9 version is at the end -> The OpenJ9 version `openj9-0.55.0` is at the end